fastlane/Fastfile

```diff
@@ -1074,31 +1085,6 @@ platform :ios do
   end
 end
 
-def fastlane_directory
-  __dir__
-end
-
-def derived_data_directory
-  File.join(fastlane_directory, 'DerivedData')
-end
-
-def screenshots_directory
-  File.join(fastlane_directory, 'screenshots')
-end
-
-def screenshot_devices
-  [
-    'iPhone 11 Pro Max',
-    'iPhone 8 Plus',
-    'iPad Pro (12.9-inch) (2nd generation)',
-    'iPad Pro (12.9-inch) (3rd generation)'
-  ]
-end
-
-def simulator_version
-  '14.4'
-end
-
```

Comment:
The values in these methods were already stored in constants at the top of the Fastfile. I verified that they aren't being used anywhere.